### PR TITLE
Did some Grammar and spelling correction

### PR DESCRIPTION
**Pull request by kartik-jobs**
## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,14 @@
 </a>
 
 <h1 align="center">
-  Welcome to Kodi Home Theater Software!
+  Welcome to Kodi Home Theatre Software!
```
Comment (DaVukovic) [inline README.md:29]: In AE (American English) the spelling is "Theater". In BE (Britain English) the Spelling is "Theatre". So there's nothing wrong while using "Theater", especially as Kodis trademark is a US trademark.

File: README.md [hunk 2]
```diff
@@ -50,11 +50,11 @@ Kodi's developers work hard to make it support a large range of devices and oper
...
+* **Coding:** Developers can help Kodi by **[fixing a bug](https://github.com/xbmc/xbmc/issues)**, adding new features, making our technology smaller and faster and making development easier for others. Kodi's codebase consists mainly of C++ with small parts are written in a variety of coding languages. Our add-ons mainly consist of python and XML. For more information, please have a look at our **[contributing guide](docs/CONTRIBUTING.md)**.
 * **Helping users:** Our support process relies on enthusiastic contributors like you to help others get the most out of Kodi. The #1 priority is always answering questions in our **[support forums](https://forum.kodi.tv/)**. Everyday new people discover Kodi, and everyday they are virtually guaranteed to have questions.
```
Comment (DaVukovic) [inline README.md:54]: true

File: README.md [hunk 3]
```diff
@@ -50,11 +50,11 @@ Kodi's developers work hard to make it support a large range of devices and oper
...
 * **Localization:** Translate **[Kodi](https://kodi.weblate.cloud/projects/kodi-core/kodi-main/)**, **[add-ons, skins etc.](https://kodi.weblate.cloud/)** into your native language.
 * **Add-ons:** **[Add-ons](https://kodi.tv/addons)** are what make Kodi the most extensible and customizable entertainment hub available. **[Get started building an add-on](https://kodi.tv/create-an-addon)**.
-* **Documentation:** Kodi's **[wiki pages](https://kodi.wiki/)** are the hub for information about Kodi and surrounding ecosystem. Help make our documentation better by writing new content or correcting existing material.
+* **Documentation:** Kodi's **[wiki pages](https://kodi.wiki/)** are the hub for information about Kodi and the surrounding ecosystem. Help make our documentation better by writing new content or correcting existing material.
```
Comment (DaVukovic) [inline README.md:58]: Not so sure about this one. I would suggest `about Kodi and its surrounding....`

**Comment by enen92** [comment]
Yeah, we got 3 already and counting... I don't think there's much we can do (not worth the effort too). They state for maintainers to set the `invalid` and `spam` labels just like I did above ^^. Ideally they should have some sort of backend service that disqualifies individuals if they have a single spam PR but I'm not sure..Highly likely that this one won't count towards the competition but others will, specially if they are able to have PRs opened for 14 days

**Pull request closed without merging**